Rest/RestTests/RestTests.csproj

```diff
@@ -1,163 +1,33 @@
   </PropertyGroup>
+
   <ItemGroup>
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\packages\Newtonsoft.Json.12.0.2\lib\net45\Newtonsoft.Json.dll</HintPath>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Net.Http" />
-    <Reference Include="System.Web" />
+    <Compile Remove="TestResults\**" />
+    <EmbeddedResource Remove="TestResults\**" />
+    <None Remove="TestResults\**" />
   </ItemGroup>
-  <Choose>
-    <When Condition="('$(VisualStudioVersion)' == '10.0' or '$(VisualStudioVersion)' == '') and '$(TargetFrameworkVersion)' == 'v3.5'">
-      <ItemGroup>
-        <Reference Include="Microsoft.VisualStudio.QualityTools.UnitTestFramework, Version=10.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
-      </ItemGroup>
-    </When>
-    <Otherwise>
-      <ItemGroup>
-        <Reference Include="Microsoft.VisualStudio.QualityTools.UnitTestFramework">
-          <Private>False</Private>
-        </Reference>
-      </ItemGroup>
-    </Otherwise>
-  </Choose>
+ 
   <ItemGroup>
-    <Compile Include="EchoSupport.cs" />
-    <Compile Include="FewTypes.cs" />
-    <Compile Include="ManyTypes.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="FitNesseFormatterTests.cs" />
-    <Compile Include="RestRequestMock.cs" />
-    <Compile Include="RestRequestMockFactory.cs" />
-    <Compile Include="RestSessionTests.cs" />
-    <Compile Include="SessionContextTests.cs" />
-    <Compile Include="JsonObjectTests.cs" />
-    <Compile Include="TextObjectTests.cs" />
-    <Compile Include="ContentHandlerTests.cs" />
-    <Compile Include="PropertiesForObjectTest.cs" />
-    <Compile Include="RestTesterTests.cs" />
-    <Compile Include="RestConfigTest.cs" />
-    <Compile Include="ContentObjectTest.cs" />
-    <Compile Include="ContentObjectFactoryTests.cs" />
-    <Compile Include="ExtensionFunctionsTests.cs" />
-    <Compile Include="ApplicationInfoTest.cs" />
-    <Compile Include="RestRequestTests.cs" />
-    <Compile Include="InjectorTests.cs" />
-    <Compile Include="XmlObjectTests.cs" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.4" />
+    <PackageReference Include="MSTest.TestAdapter" Version="2.2.3" />
+    <PackageReference Include="MSTest.TestFramework" Version="2.2.3" />
+    <PackageReference Include="coverlet.collector" Version="3.0.2" />
   </ItemGroup>
+
+ 
   <ItemGroup>
-    <ProjectReference Include="..\Rest\Rest.csproj">
-      <Project>{0BA5DBEB-3B3C-4BA7-B5FC-6719F23B3175}</Project>
-      <Name>Rest</Name>
-    </ProjectReference>
+    <ProjectReference Include="..\Rest\Rest.csproj" />
   </ItemGroup>
+
+ 
   <ItemGroup>
-    <Content Include="TestData.xml">
+    <None Update="TestData.xml">
```

Comment:
If you moved the match parsing test data to code, this file should not be required anymore.